build.sh

```diff
@@ -308,11 +308,11 @@ if [ "$BUILD_TARGET" = "dist" ] ; then
         OPTIONS+=(-v)
     fi
 
-    OPTIONS+=(-f $SRC_DIR/build.xml)
-    OPTIONS+=(-Dversion=$VERSION)
-    OPTIONS+=(-Djnidir=$JNI_DIR)
-    OPTIONS+=(-Dsrc.dir=tomcat-$TOMCAT_VERSION)
-    OPTIONS+=(-Dbuild.dir=$WORK_DIR)
+    OPTIONS+=(-f "$SRC_DIR"/build.xml)
+    OPTIONS+=(-Dversion="$VERSION")
+    OPTIONS+=(-Djnidir="$JNI_DIR")
+    OPTIONS+=(-Dsrc.dir=tomcat-"$TOMCAT_VERSION")
+    OPTIONS+=(-Dbuild.dir="$WORK_DIR")
```

Comment:
I would have tough they should be like:

```
OPTIONS+=("-f ${SRC_DIR}/build.xml")
```
using the ${} format.

Reply:
I used this reference: https://www.shellcheck.net/wiki/SC2206

I'm no shell expert, I'm learning as I'm going here, so I'm pleased to be getting feedback so I can learn :+1:

Reply:
In the reference the option has only the $var and it is quoted to be a single string. In your case you have multiple strings so the first option are two. Not a big issue anyway

Reply:
I would think that in general we should be quoting the whole value/string, not just the variables, so in this case:
```
OPTIONS+=(-f "$SRC_DIR/build.xml")
```
I think the above code is adding two values to the `OPTIONS` array: the `-f` and the `$SRC_DIR/build.xml`, so they should not be enclosed in the same quotes.

In the line below I think it should be like this:
```
OPTIONS+=(-Dsrc.dir="tomcat-$TOMCAT_VERSION")
```
or
```
OPTIONS+=("-Dsrc.dir=tomcat-$TOMCAT_VERSION")
```
The first one makes more sense to me, but either one probably works.

Reply:
OK, pushed with suggested changes.

Reply:
Thanks for the update! BTW, it might be OK to separate `-D` from the `<name>=<value>` pair, but I think usually they're concatenated. That's a different topic though.